src/utils/findProvidesModule.js

```diff
@@ -57,7 +57,7 @@ function findProvidesModule(directories, opts = {}) {
   const modulesMap = {};
 
   const walk = dir => {
-    const stat = fs.lstatSync(dir);
+    const stat = fs.statSync(dir);
```

Comment:
This fixes an issue with having symlinks within the react-native package, which some package managers end up doing in some cases.

Reply:
No sure about this change: https://github.com/callstack/haul/pull/171

Reply:
Looks like this was a bug fixed in RN 0.47: https://github.com/facebook/react-native/commit/7d1981ef8245cd6267823a970054cbebb2d2fd14